presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -16,6 +16,8 @@
 import com.facebook.presto.client.ClientSession;
 import com.facebook.presto.client.ServerInfo;
 import com.facebook.presto.client.StatementClient;
+import com.facebook.presto.spi.security.SelectedRole;
```

Comment:
The change in this file is slightly different from https://github.com/prestosql/presto/pull/90/commits/e02bdabfe6a7bea3459b4d3147f36fe85f3554d3

Reply:
https://github.com/prestodb/presto/pull/12383#issuecomment-467633410